srcpkgs/gnome-flashback/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'gnome-flashback'
+pkgname=gnome-flashback
+version=3.30.0
+revision=1
+only_for_archs="i686 x86_64"
+build_style=gnu-configure
+configure_args="--prefix=/usr --sysconfdir=/etc --localstatedir=/var --libexecdir=/usr/lib/${pkgname} --disable-schemas-compile --enable-compile-warnings=minimum --enable-gtk-doc"
+hostmakedepends="automake autogen autoconf-archive gettext-devel intltool libtool git gobject-introspection gtk-doc intltool libxslt pkg-config
+python3 sassc"
+makedepends="caribou-devel folks-devel
+gjs-devel gnome-bluetooth-devel gnome-control-center-devel gnome-desktop-devel
+gnome-menus-devel ibus-devel libcanberra-devel libcroco-devel mutter-devel
+network-manager-applet-devel polkit-devel pulseaudio-devel
+startup-notification-devel telepathy-logger-devel upower-devel"
+depends="gnome-bluetooth gnome-session gnome-settings-daemon gnome-themes-standard"
+short_desc="GNOME 3 session with similar user experience to the GNOME 2.x series"
+maintainer="shizonic <realtiaz@gmail.com>"
+license="GPL-3.0"
+homepage="https://wiki.gnome.org/Projects/GnomeFlashback"
+
+do_fetch() {
+	git clone --recursive https://gitlab.gnome.org/GNOME/gnome-flashback.git ${wrksrc}
```

Comment:
We don't use git except where unavoidable. The version's tarball can be downloaded at
https://gitlab.gnome.org/GNOME/gnome-flashback/-/archive/3.30.0/gnome-flashback-3.30.0.tar.bz2
so you should use
`distfiles="https://gitlab.gnome.org/GNOME/gnome-flashback/-/archive/${version}/${pkgname}-${version}.tar.bz2"`
and set the checksum
`checksum=f4015a7e87ba3b889b486431087ea91831b5a314dbc888ddef0b2f4ad22e582e`